Gemfile

```diff
@@ -131,7 +130,7 @@ group :test, :development do
   gem 'awesome_print'
   gem 'capybara', '>= 2.18.0' # 3.0 requires nokogiri 1.8
   gem 'database_cleaner', require: false
-  gem "factory_bot_rails", '4.8.2', require: false
+  gem "factory_bot_rails", '4.10.0', require: false
```

Comment:
is there a particular reason why we can use only 4.10.0 and not any patches to come? Just wondering if we can set the version a bit looser so that dependabot fins no trouble upgrading it for us.

Reply:
I just followed your advise:
https://github.com/openfoodfoundation/openfoodnetwork/pull/4930#discussion_r392170518
we will be fine until rails 4.2 anyway.

Reply:
My point is that in this case, we can do `~> 4.10` so running a `bundle update factory_rails_bot` will pick up any patches on the 4.10 series.